README.md

```diff
@@ -68,117 +99,85 @@ nslookup ${He_Name}.azurecr.io
 
 Create Resource Groups
 
-- When experimenting with this app, you should create new resource groups to avoid accidentally deleting resources
-
-  - If you use an existing resource group, please make sure to apply resource locks to avoid accidentally deleting resources
+> When experimenting with this app, you should create new resource groups to avoid accidentally deleting resources
+>
+> If you use an existing resource group, please make sure to apply resource locks to avoid accidentally deleting resources
 
 - You will create 3 resource groups
   - One for ACR
   - One for App Service or AKS, Key Vault and Azure Monitor
-  - One for Cosmos DB (see create and load sample data to Cosmos DB step)
+  - One for Cosmos DB
 
 ```bash
 
 # set location
-export He_Location=centralus
+export He_Location=
+
 
-# resource group names
+# set resource group names
 export He_ACR_RG=${He_Name}-rg-acr
 export He_App_RG=${He_Name}-rg-app
 
 # create the resource groups
+# the Cosmos DB resource group is created later
 az group create -n $He_App_RG -l $He_Location
 az group create -n $He_ACR_RG -l $He_Location
 
-```
-
-Save your environment variables for ease of reuse and picking up where you left off.
-
-```bash
-
-# run the saveenv.sh script at any time to save He_*, Imdb_*, MSI_*, and AKS_* variables to ~/${He_Name}.env
-# make sure you are in the root of the repo
+# run the saveenv.sh script at any time to save He_*, Imdb_*, MSI_*, and AKS_* variables to ~/.helium.env
 ./saveenv.sh
```

Comment:
We should call out somewhere (maybe here) that the saveenv.sh script also sets the He_Repo variable so there's somewhere to reference when it's used later in the readme

Reply:
done